x-pack/plugins/security_solution/server/lib/detection_engine/routes/__mocks__/request_context.ts

```diff
@@ -8,10 +8,13 @@
 import type { MockedKeys } from '@kbn/utility-types/jest';
 import { coreMock } from 'src/core/server/mocks';
 
-import { ActionsApiRequestHandlerContext } from '../../../../../../actions/server';
-import { AlertingApiRequestHandlerContext } from '../../../../../../alerting/server';
+import type { ActionsApiRequestHandlerContext } from '../../../../../../actions/server';
+import type { AlertingApiRequestHandlerContext } from '../../../../../../alerting/server';
 import { rulesClientMock } from '../../../../../../alerting/server/mocks';
-import { actionsClientMock } from '../../../../../../actions/server/mocks';
+
+// We have to disable this path otherwise we leak memory and cannot directly import from "../../../../../../actions/server/"
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { actionsClientMock } from '../../../../../../actions/server/actions_client.mock';
```

Comment:
NOTE: I have to use this and cannot use `import { actionsClientMock } from '../../../../../../actions/server/mocks';` without having a memory leak within Jest.

Reply:
Is there anyway we can add a linter rule to this and the above infringing mocks to prevent folks from importing them this way in the future (or warn at least?), or best just to prioritize identifying and fixing the underlying issues? Just curious what we can do to prevent these memory leaks from slowly sneaking their way back in... 🐌💧

Reply:
I am thinking that maybe this isn't the right way for us to go here. I am thinking of putting this back into draft mode and talking to more operations people. The module loader in some cases looks to be re-loading or leaking the modules and handling this at an infrastructure level might be better for us.